### DISCUSSION TITLE
Umi build命令生成.umi文件，控制台报错

### DISCUSSION THREAD

**Discussion by myy-1** [umijs/umi#12490] [Q&A]
![image](https://github.com/umijs/umi/assets/71581579/22603666-3cbc-4049-85a9-20f7a4435c7b)
命令行新建的Umi项目，怎样会让.umi下自动生成并带上

**Comment by myy-1** [comment]
![image](https://github.com/umijs/umi/assets/71581579/6322a73c-c8bc-473c-a11b-4c08fb80172f)
如何通过命令行新建的Umi项目，怎样会让.umi下自动生成并带上上述截图的文件夹？

**Comment by fz6m** [comment]
umi 4 有两种初始模板，一个是 umi ，一个是 max ，umi 是最小的模板，不自带任何其他额外的功能（如 qiankun / model / mf 等），如果在 umi 项目上要用这些功能，需要显示安装 `@umijs/plugins` 并按需配置，否则，请使用 max 模板，这个模板自带了所有的 `@umijs/plugins` ，无需再安装了。

详见 [插件](https://umijs.org/docs/guides/use-plugins) 文档。
